js/pages/samples.js

```diff
@@ -588,6 +614,14 @@ const renderLocations = (site) => {
                         </div>
                     </div>
                 </div>`
+            } else if (site === u_chicago && !location[1]){
+                template += `<div class="row" style="width:100%">
```

Comment:
What exactly is this code doing? If `location[1]` is a falsy value, we want to render it?

Reply:
This condition is for the content under the "Where do I donate My Samples" header. To remove the subheading "address and directions" under this header I added that piece of code. But Address and locations will be visible for all the locations.